packages/flutter_tools/templates/ffi_plugin/macos.tmpl/projectName.podspec.tmpl

```diff
@@ -0,0 +1,22 @@
+#
```

Comment:
as above, this file should be documented in detail (assume the reader barely knows dart and certainly doesn't know Pod)